src/test/scala/scala/compat/java8/StepConvertersTest.scala

```diff
@@ -0,0 +1,572 @@
+    Okay( cm.AnyRefMap[String,String]("fish" -> "salmon").valueStepper )
+    good( cm.ArrayBuffer[String]("salmon").stepper )
+    good( ((Array("salmon"): cm.WrappedArray[String]): cm.ArrayLike[String, cm.WrappedArray[String]]).stepper )
+    good( (Array("salmon"): cm.ArrayOps[String]).stepper )
+    good( cm.ArraySeq[String]("salmon").stepper )
+    _eh_( cm.ArrayStack[String]("salmon").stepper )
+    Okay( (cm.ArrayBuffer[String]("salmon"): cm.Buffer[String]).stepper )
+    good( cm.HashMap[String, String]("fish" -> "salmon").stepper )
+    good( cm.HashMap[String, String]("fish" -> "salmon").keyStepper )
+    good( cm.HashMap[String, String]("fish" -> "salmon").valueStepper )
+    good( cm.HashSet[String]("salmon").stepper )
+    good( cm.IndexedSeq[String]("salmon").stepper )
+    good( cm.IndexedSeq[String]("salmon").view.stepper )
+    Okay( cm.Iterable[String]("salmon").stepper )
+    Okay( cm.LinearSeq[String]("salmon").stepper )
+    good( cm.LinkedHashMap[String, String]("fish" -> "salmon").stepper )
+    good( cm.LinkedHashMap[String, String]("fish" -> "salmon").keyStepper )
+    good( cm.LinkedHashMap[String, String]("fish" -> "salmon").valueStepper )
+    Okay( cm.LinkedHashSet[String]("salmon").stepper )
+    Okay( cm.ListBuffer[String]("salmon").stepper )
+    Okay( cm.ListMap[String, String]("fish" -> "salmon").stepper )
+    Okay( cm.ListMap[String, String]("fish" -> "salmon").keyStepper )
+    Okay( cm.ListMap[String, String]("fish" -> "salmon").keyStepper )
+    Okay( cm.LongMap[String](9876543210L -> "salmon").stepper )
+    Okay( cm.LongMap[String](9876543210L -> "salmon").valueStepper )
+    Okay( cm.Map[String, String]("fish" -> "salmon").stepper )
+    Okay( cm.Map[String, String]("fish" -> "salmon").keyStepper )
+    Okay( cm.Map[String, String]("fish" -> "salmon").valueStepper )
+    Okay( cm.MutableList[String]("salmon").stepper )
+    Okay( cm.OpenHashMap[String, String]("fish" -> "salmon").stepper )
+    Okay( cm.OpenHashMap[String, String]("fish" -> "salmon").keyStepper )
+    Okay( cm.OpenHashMap[String, String]("fish" -> "salmon").valueStepper )
+    Okay( cm.PriorityQueue[String]("salmon").stepper )
+    Okay( cm.Queue[String]("salmon").stepper )
+    good( cm.ResizableArray[String]("salmon").stepper )
+    Okay( cm.Seq[String]("salmon").stepper )
+    Okay( cm.Set[String]("salmon").stepper )
+    Okay( cm.SortedSet[String]("salmon").stepper )
+    Okay( cm.Stack[String]("salmon").stepper )
+    IFFY( cm.Traversable[String]("salmon").accumulate.stepper )
+    Okay( cm.TreeSet[String]("salmon").stepper )
+    Okay( cm.UnrolledBuffer[String]("salmon").stepper )
+    Okay( cm.WeakHashMap[String, String]("fish" -> "salmon").stepper )
+    Okay( cm.WeakHashMap[String, String]("fish" -> "salmon").keyStepper )
+    Okay( cm.WeakHashMap[String, String]("fish" -> "salmon").valueStepper )
+    good( (Array("salmon"): cm.WrappedArray[String]).stepper )
+
+    // Java 6 converters section
```

Comment:
I think I was supposed to write wrappers for the pre-Java-8 collections wrappers.  And I don't think I did.